package.json

```diff
@@ -15,7 +15,8 @@
     "format": "prettier --write \"./**/*.{ts,tsx}\"",
     "lint": "eslint \"**/*.{ts,tsx}\" \"*.config.js\"",
     "test": "npm run type-check && npm run lint",
-    "type-check": "tsc --noEmit"
+    "type-check": "tsc --noEmit",
+    "validate": "npm run lint && npm run type-check"
```

Comment:
Do we really need to run the lint and type check on the CI, I think we can use husky and pre-commit?

Reply:
We need it, because Git pre-commit hook can simply be skipped by using `--no-verify` flag. 😅